app/lib/ThangTypeConstants.coffee

```diff
@@ -37,7 +37,7 @@ ThangTypeConstants =
     'hero-e': '60ff9bc75b55bb002971ba41'  # Female, androgynous
     'hero-f': '60ff9bd728067a00236eaa57'  # Male, wheelchair
   heroClasses:
-    Warrior: ['champion', 'duelist', 'captain', 'knight', 'samurai', 'raider', 'goliath', 'guardian', 'code-ninja', 'stalwart', 'armando-hoyos']
+    Warrior: ['champion', 'duelist', 'captain', 'knight', 'samurai', 'raider', 'goliath', 'guardian', 'code-ninja', 'stalwart', 'armando-hoyos', 'ned-fulmer']
```

Comment:
i think we are removing it, I remember @Chaboi45 working on it!!

Reply:
Should still be classified correctly for legacy uses of the hero. Actually, that removal paradoxically made Ned _more_ available, because it removed the you-have-to-be-in-this-clan-to-get-this-warrior exception, and every student got him. Fixing in the next commit.